### PR TITLE
Alignment of ODS and Reporting scripts(appointments linelist). Modifi…

### PR DESCRIPTION
**Pull request by DennisGibz**
…ed some NDWH scripts and folders

### CODE REVIEW

**Comment by nobert-mumo** [comment]
@DennisGibz I see the PR is quite big, 39 files changed; Mayne you can add in the description more context on the changes to enable reviewers give feedback.

**Comment by DennisGibz** [comment]
1. Load CT_Patient
      - Removed voided = 0 from the inner query which helps in removing the Dups
2. Load Fact ART
 - Included a script to load the data into factARTHistory
3.ODS,HTS,MNCH & PrEP tables
  - casted the Dates tables(Created and ModifiedDates) in the inner quieries.
 - Included Date_last_modified in the join to cater fore some duplicates
 -

File: Scripts/REPORTING/static/load_linelist_appointments.sql
Comment (nobert-mumo) [inline Scripts/REPORTING/static/load_linelist_appointments.sql:1]: @DennisGibz this was intentional

File: Scripts/NDWH/C&T FACT TABLES/load_FactART.sql [hunk 1]
```diff
@@ -1,3 +1,19 @@
+---------------Insert into FactArtHistory before droping FactART for new data
+BEGIN
+	INSERT INTO [NDWH].[dbo].[FactARTHistory]([FactKey],[FacilityKey],[PartnerKey],[AgencyKey],[PatientKey],[AsOfDateKey],[IsTXCurr],
+												[ARTOutcomeKey],[NextAppointmentDate],[LastEncounterDate],[LoadDate],DateTimeStamp)
+	SELECT [FactKey],[FacilityKey],[PartnerKey],[AgencyKey],[PatientKey],NULL [AsOfDateKey],
+		CASE
```
[the remark this answers is not in the document]
Comment (DennisGibz) [reply Scripts/NDWH/C&T FACT TABLES/load_FactART.sql]: @nobert-mumo thanks for highlighting above. Necessary corrections have been done accordingly. Thank you.

**Review by nobert-mumo — APPROVED** [review]
LGTM

**Pull request merged**